src/main/java/io/smartcat/migration/MigrationEngine.java

```diff
@@ -94,6 +104,16 @@ public boolean migrate(final MigrationResources resources) {
 
             return true;
         }
+
+        /**
+         * Method that checks if all cluster nodes are up and healthy by getting current migration type with consistency level ALL. This
```

Comment:
This comment isn't true. With consistency level ALL the client will wait for all replicas to respond. With a typical replication factor of 3 on the keyspace containing schema version table it will require all 3 nodes that contain the data to respond. It doesn't necessarily mean it will require all nodes to be up in a cluster larger than 3 nodes.